client/ayon_maya/plugins/publish/collect_fbx_animation.py

```diff
@@ -34,3 +42,33 @@ def process(self, instance):
                 ))
             if skeleton_content:
                 instance.data["animated_skeleton"] = skeleton_content
+
+        attribute_values = self.get_attr_values_from_data(
+            instance.data
+        )
+
+        instance.data["upAxis"] = attribute_values.get(
+            "upAxis", self.up_axis)
+        instance.data["inputConnections"] = attribute_values.get(
+            "inputConnections", self.input_connections)
+
+    @classmethod
+    def get_attribute_defs(cls):
+        defs = [
+            UISeparatorDef("sep_fbx_options"),
+            UILabelDef("Fbx Options"),
+        ]
+        defs.extend(
+            super(CollectFbxAnimation, cls).get_attribute_defs() + [
+            EnumDef("upAxis",
+                    ["x", "y", "z"],
+                    default=cls.up_axis,
+                    tooltip="Define the up axis in your FBX file"),
+            BoolDef("inputConnections",
+                    default=cls.input_connections,
+                    tooltip="Whether the input connections to "
+                            "selected object to be exported"),
```

Comment:
Odd english grammar.
```suggestion
                    tooltip=(
                        "Whether to include input connections to "
                        "selected objects to be exported."
                    ),
```

Or staying closer to what you had:
```suggestion
                    tooltip=(
                        "Whether input connections to "
                        "selected objects are to be exported."
                    ),
```